src/steps/installs/index.test.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  createMockStepExecutionContext,
+  Recording,
+} from '@jupiterone/integration-sdk-testing';
+import { IntegrationConfig } from '../../config';
+import { integrationConfig } from '../../../test/config';
+import { setupWPEngineRecording } from '../../../test/recording';
+import { fetchInstalls } from '.';
+import { fetchSites } from '../sites';
+import { fetchAccounts } from '../accounts';
+
+describe('#fetchSite', () => {
+  let recording: Recording;
+
+  beforeEach(() => {
+    recording = setupWPEngineRecording({
+      directory: __dirname,
+      name: 'fetchSite',
+    });
+  });
```

Comment:
You forgot to change the name here, should be `fetchInstalls` :slightly_smiling_face: